### PR TITLE
[Test ] Update to Webpack 5 for video test app

### PR DESCRIPTION
**Pull request by ltrung**
**Description of changes:**
- Update webpack config for video integration test app to Webpack 5.
- Add video test app tests to GitHub Action

**Testing**

1. Have you successfully run `npm run build:release` locally? N/A
2. How did you test these changes? Manually run the video test app
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it? N/A
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? N/A
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: integration/js/script/run-test [hunk 1]
```diff
@@ -123,6 +133,10 @@ case $1 in
       r meeting_leave_video_test.config.json
       r reconnection_test.config.json
       ;;
+   video_test_app)
+      r test_app_video_test.config.json
+      r test_app_video_pin_test.config.json
```
Comment (michhyun1) [inline integration/js/script/run-test:138]: We eventually want to use the test app for the video E2E test too

File: package.json [hunk 2]
```diff
@@ -44,7 +44,8 @@
     "test:integration-content-share": "cd ./integration/js && npm run test -- content_share ",
     "test:integration-data-message": "cd ./integration/js && npm run test -- data_message ",
     "test:integration-meeting-readiness-checker": "cd ./integration/js && npm run test -- meeting_readiness_checker ",
-    "test:integration-messaging": "cd ./integration/js && npm run test -- messaging "
+    "test:integration-messaging": "cd ./integration/js && npm run test -- messaging ",
+    "test:integration-video-test-app": "cd ./integration/js && npm run test -- video_test_app "
```
Comment (michhyun1) [inline package.json:48]: cool, thanks!

**Review by michhyun1 — COMMENTED** [review]
thanks for adding this!

**Review by michhyun1 — APPROVED** [review]

File: integration/js/script/run-test (hunk 1 above)
Comment (ltrung) [reply integration/js/script/run-test:138]: it didn't work when i added it yesterday so need to check that later.

**Pull request merged**